metadata-models/src/main/pegasus/com/linkedin/common/RoleAssociation.pdl

```diff
@@ -0,0 +1,17 @@
+namespace com.linkedin.common
+
+
+@Aspect = {
+  "name": "roleAssociation"
+}
+record RoleAssociation {
+
+ /**
+  * Urn of the applied External Role
+  */
+  @Relationship = {
+    "name": "AssociatedWith",
+    "entityTypes": [ "role" ]
+  }
+  urn: RoleUrn
```

Comment:
This can just be "Urn", we recommend against using strongly-typed urns now. (they are legacy)

Reply:
Done